README.md

```diff
@@ -200,7 +200,7 @@ truecolor either.
   {real truecolors} available) - **Windows platform**
 - [Tera Term](http://en.sourceforge.jp/projects/ttssh2/) [delimiter: colon,
   semicolon] - **Windows platform**
-- [ConEmu](https://github.com/Maximus5/ConEmu) [delimiter: semicolon] -
+- [ConEmu](https://github.com/Maximus5/ConEmu) [delimiter: semicolon] - written in Rust
```

Comment:
Last time I checked it certainly wasn't 😆

Reply:
Weird, it doesn't seemed like this is written in rust, I think I pasted wrongly.